drivers/wdm/audio/backpln/portcls/miniport.cpp

```diff
@@ -40,6 +40,15 @@ operator delete(
     ExFreePool(ptr);
 }
 
+void
+__cdecl
+operator delete(PVOID ptr,
+                UINT64 ,
+                enum std::align_val_t)
```

Comment:
Following ambient code-style:
```suggestion
operator delete(
    PVOID ptr, UINT64,
    enum std::align_val_t)
```

Also, is it really UINT64 here? or UINT_PTR (i.e. 32-bit long on 32-bit builds, and 64-bit long on 64-bit ones) as in the two other functions above?